### DISCUSSION TITLE
Streamline READMEs

### DISCUSSION THREAD

**Discussion by ShafSpecs** [ShafSpecs/remix-pwa#91] [Ideas]
Currently, the README of the various packages are out of sync including versions. For example, when `sw` package is updated, `remix-pwa` isn't and so on. 

Ideas:
- Streamline the packages so that they always pull the latest feature
- Streamline the packages and link their correct documentation
- Create changelogs for each package to accurately keep track of changes across the major/minor versions 
- Suggestion: Add the changelog feature to the doc site

**Comment by dev-afzalansari** [comment]
> Streamline the packages so that they always pull the latest feature

Does it mean all packages will have the same version as `remix-pwa`? For e.g. `remix-pwa` -> `2.0.0`, `@remix-pwa/sw` -> `2.0.0`, `@remix-pwa/push` -> `2.0.0`.

**Reply by ShafSpecs** [reply]
That's a possibility. I am not sure how it's going to work tho, but yes, that's a real possibility
